src/applications/office-directory/containers/App.jsx

```diff
@@ -50,47 +27,22 @@ export default function App() {
     getJson();
   }, []);
 
-  useEffect(
-    () => {
-      if (filterTopOffices) {
-        setDisplayOffices(
-          allOffices.filter(office => topOffices.includes(office.entityId)),
-        );
-      } else {
-        setDisplayOffices(allOffices);
-      }
-    },
-    [allOffices, filterTopOffices],
-  );
-
-  const toggleFilterTopOffices = useCallback(() => {
-    setFilterTopOffices(prevValue => !prevValue);
-  }, []);
-
   return (
     <div className="vads-l-grid-container vads-u-padding-x--1p5 vads-u-padding-bottom--4">
       <h1 className="vads-u-margin-bottom--0">All VA offices</h1>
 
-      <input
+      {/* <input
```

Comment:
This commented code can be removed. I hope to never see this code of mine back in here!